meson.build

```diff
@@ -256,7 +256,7 @@ if get_option('tests').enabled()
         'beancount/parser/decimal.c',
         'beancount/parser/tokens.c',
         'beancount/parser/tokens_test.c',
-        dependencies: [libm, py.dependency(embed: true)],
+        dependencies: [py.dependency(embed: true)],
```

Comment:
Why this change?

`libm` is needed on some systems and Meson should do the right thing on systems where it is not.